bionic/core/flow_execution.py

```diff
@@ -67,11 +68,15 @@ def __init__(self, bootstrap, flow_instance_uuid, task_key_logger):
 
     @property
     def _parallel_execution_enabled(self):
-        return self._bootstrap.process_executor if self._bootstrap is not None else None
+        if self._bootstrap is None:
+            return False
+        return self._bootstrap.process_executor is not None
```

Comment:
What about `return self._bootstrap is not None and self._bootstrap.process_executor is not None` ?